mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule UeberauthGoogle.Mixfile do
   defp deps do
     [
      {:oauth2, "~> 1.0.0"},
-     {:ueberauth, "~> 0.6.0"},
+     {:ueberauth, "~> 0.6.0", git: "https://github.com/foxtrod/ueberauth.git"},
```

Comment:
Can you please put the official `ueberauth` dep here?

Reply:
@tajchumber Sure, but I got this pending [PR](https://github.com/ueberauth/ueberauth/pull/108) :). Is it better to change mix after merging of PR which im waiting for, or just ping you to merge when they will apply it?

Reply:
@foxtrod we'll have to wait for the other PR to be accepted first.
I will look into it shortly! :)